client/src/components/body/landingpage/index.js

```diff
@@ -187,6 +188,37 @@ const Home = (props) => {
               </div>
             </div>
             </div>
+            <div class="row">
+              <div
+                  className="col-md-4 portfolio-item">
+                <a
+                    className="portfolio-link"
+                    onClick={() => handleRedirect(actions, 5)}
+                    href="# "
+                >
+                  <img
+                      className="img-fluid landingpage__image"
+                      src={cognitive}
```

Comment:
The image looks distorted on the landing page - I think cropping the original image to fit square dimensions could fix this